.github/ISSUE_TEMPLATE/issue_template.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Issue template
+about: Define de structure of a task. Help us to be more specific. Avoid the incertitude
+title: ''
+labels: enhancement
+assignees: ''
+
+---
+
+## Journey or Title
+For titles use the format is 'As a <ROLE>, i want to <GOAL>, so that <REASON>'
+For journeys  add a little description of what's actually happen and what we expected to happen
+
+## Acceptance criteria
+- Describe, how we want the "what"
+- What are the specifications to be accepted
```

Comment:
What do you think if we rephrase to this: 
`What are the specifications that must be met`